README.md

```diff
@@ -171,20 +171,35 @@ leptos_hotkeys = { version = "0.1.6", features = ["hydrate"] }
 
 We also other feature flags that enhance developer experience, to learn more read [feature-flags](#feature-flags)
 
-### Hotkey Provider
+### `provide_hotkeys_context()`
 
-Wrap your project with `<HotkeysProvider />`:
+Call `provide_hotkeys_context()` in the `App()` component. This will provide the `HotkeysContext` for the current reactive node and all of its descendents. 
+This function takes three parameters, the `node_ref`, a flag to disable blur events (*you should go `false`*), and a list of `initially_active_scopes`.
 
-```html
-view! {
-    <HotkeysProvider>
-        <Router>
-            <Routes>
-                <Route path="/" view=HomePage/>
-                <Route path="/:else" view=ErrorPage/>
-            </Routes>
-        </Router>
-    </HotkeysProvider>
+> [!NOTE]
+>
+> `provide_hotkeys_context()` returns a `HotkeysContext`. To learn more, see [HotkeysContext](#hotkeyscontext).
+
+```rust
+#[component]
+pub fn App() -> impl IntoView {
+    provide_meta_context();
+    
+    let main_ref = create_node_ref::<html::Main>();
+    provide_hotkeys_context(main_ref, false, scopes!()); 
```

Comment:
Additionally, I would expect that `provide_hotkeys_context` take a new type `ProvideHotkeysContextOptions` that would implement `From` for `NodeRef` (or better maybe signal to noderef), `bool`, tuple of both or whatever builds `scopes!`. With that, I could use `provide_hotkeys_context(main_ref)` directly and `provide_hotkeys_context((main_ref, false))` or better `ProvideHotkeysContextOptions::new(main_ref).with_allow_blur_event(true)`... etc.

But this is surely something that can be implemented later. I don't know if you follow me, if not, let me know.